src/libs/OptionsListUtils.ts

```diff
@@ -2491,6 +2491,11 @@ function filterOptions(options: Options, searchInputValue: string, config?: Filt
                 values = values.concat(getParticipantsLoginsArray(item));
             }
 
+            const partialReport = {reportID: item.reportID, type: item.type, chatType: item.chatType};
+            if (!ReportUtils.isChatReport(partialReport) || !ReportUtils.isChatRoom(partialReport)) {
```

Comment:
Why are we doing this `partialReport` thing?

Reply:
Because those are the only fields needed to pass into `isChatReport` and `isChatRoom` and are also the only fields that we have access to in the `item` object.

It's either this, or we cache them in `result`, which we decided we didn't really like.

Reply:
I see we already have `item.isChatRoom` to use. 

But really I'm also not sure about this condition:

```js
if (!ReportUtils.isChatReport(partialReport) || !ReportUtils.isChatRoom(partialReport)) {
```

If it's "not a chat report" or it's "not a chat room" then we overwrite any of the logic above? Seems incorrect?

Reply:
Ah yeah I see how the logic is confusing–I fixed it by just getting rid of that else block since we want this if-block to be checked for all report types.

The reason I'm doing that is because we still want policy expense chats to be searchable by participant, so I can't include an else if.